ocserv-install.sh

```diff
@@ -62,9 +66,12 @@ read username
 
 ocpasswd -c /etc/ocserv/ocpasswd $username
 iptables -t nat -A POSTROUTING -j MASQUERADE
-sed -i -e 's@#net.ipv4.ip_forward=@net.ipv4.ip_forward=@g' /etc/sysctl.conf
+#sed -i -e 's@#net.ipv4.ip_forward=@net.ipv4.ip_forward=@g' /etc/sysctl.conf
+#The config file might be different in different OS. The command below works for debain 6.12.48.
```

Comment:
instead of making this specific and customized for Debian, let's find out the distro and decide based on that